batavia/types/Type.js

```diff
@@ -22,14 +22,19 @@ batavia.types.Object = function() {
  *************************************************************************/
 
 batavia.types.Type = function() {
-    function Type(name, bases, dict) {
+    function Type(name, bases, dict, klass) {
```

Comment:
I think this was leftover from a failed attempt. I will remove it and the corresponding changes :)